tests/org.eclipse.n4js.xpect.tests/model/typesystem/DynamicTypes.n4js.xt

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Regarding the file location and name: there are already tests for dynamic types located in `org.eclipse.n4js.spec.tests/xpect-tests/Ch04_09__TypeModifiers`.

Proposal: move/rename this new test file to `org.eclipse.n4js.spec.tests/xpect-tests/Ch04_09__TypeModifiers/DynamicTypes_subtyping.n4js.xt`.